kubernetes/samples/scripts/create-weblogic-domain-pv-pvc/README.md

```diff
@@ -213,3 +213,8 @@ Access Modes:  RWX
 Events:        <none>
 
 ```
+
+## Troubleshooting
+
+1. Message: "[ERROR] The weblogicDomainStoragePath parameter in kubernetes/samples/scripts/create-weblogic-domain-pv-pvc/create-pv-pvc-inputs.yaml is missing, null or empty"  
+Edit the file and set the value of the field.  This value must be a world-writable directory
```

Comment:
Technically I wonder if world writable is required - it perhaps only needs to be writable for the uid/gid of the pod process.

Reply:
yes, but we do not know what that uid/gid value is... that makes it a bit tougher to say than make sure it is world-writable... how would the user be able to determine that value?

Reply:
Googled a bit, and found a way to run a 'one off' command in an image via kubectl - 
```
kubectl run one-off --rm --restart=Never -it --image=store/oracle/weblogic:12.2.1.3 -- id    
uid=1000(oracle) gid=1000(oracle) groups=1000(oracle)
```

Reply:
You can set user/uid/gid when creating an image.  You can override uid/gid via annotations in the yaml (although I don't know of domain resource yaml exposes this field)...

Reply:
umm. do we need to add this one-off command to the steps executed by the customer to run the samples? Do we do something like this when we create our images, explicitly, or is uid=1000 just some sort of happy accident... My concern is this: when we document that the directory must be owned-by 1000:1000, the uid and/or gid will change to 1002 or some-such. Hilarity will ensue.

Reply:
I think it's important to point out that wide open security permissions aren't required - it's just for demo purposes.   The security conscious need to know about user/gid/uid.

Reply:
ok. I can change the instructions to use 1000:1000